### DISCUSSION TITLE
downgrade from v130

### DISCUSSION THREAD

**Discussion by buddyted** [3kh0/ext-remover#1242] [Help JOIN TITANIUM NETWORK TO SHARE PLEASE] [answered]
so my freind wants to downgrade from
 v130
 kernver 4 
board octopus 
is there anyway to downgrade to at least v125 or possibly earlier

**Comment by schoolexploitkid** [comment, marked as answer]
1. download v125-v128 recovery image CORRESPONDING to your board/codename (octopus) from https://chrome100.dev/ or https://cros.tech/
2. get a usb (if u dont have one then go cry)
3. download the chromebook recovery utility extension from chrome web store
4. plug in the usb
5. click the chromebook recovery utility extension
6. click the gear
7. click use local image
8. select your downloaded image from step 1
9. select the usb that is plugged in
10. wait until its finished
11. after its finished, unplug the usb
12. open your chromebook
13. press esc + power + refresh/reload to enter recovery
14. plug in the usb
15. wait for it to finish recovering
16. your done :P
